### PR TITLE
Adds digitigrade jumpsuit alts using ThatGuyPhil's icons

### PR DESCRIPTION
**Pull request by nmajask**
# Github documenting your Pull Request

While looking through some of the old PR's, I stumbled into "[uniform sprites for digitigrade lizardpeople #10931](https://github.com/yogstation13/Yogstation/pull/10931)" by [ThatGuyPhil](https://github.com/ThatGuyPhil). So I readded it and made it kinda work.

I have updated and added a few new icons that weren't in the original PR, but there are still **many** uniforms that need digi alts (Any that are in yogstation/icons/mob/ have not been touched). I will try to continue adding/updating important ones and will add any given to me of they work. The code is not very good, but it does function.

Due to the scope of this PR, I have done very limited testing and there are likely many issues that I have not found.

Here is an example of it working (It was taken before I fixed some stuff):
digicap

Here are some newer image:
digiminer

digisec

I have done some more testing and have tried some things that I thought would break the code. Luckily I found much fewer errors than I thought I would, for example I thought wearing a hardsuit or wearing a uniform casually would cause major issue but they didn't. The current list of issues I have found are:

- Suits still don't have digi icons or implementation, so hardsuits or any other suit that covers the wearers legs will squish them into normal legs
- Some icons have icons that look weird when wearing footwraps (medical jumpsuit is a good example
- In order to allow a uniform to use digi sprites, you need to redefine 2-3 vars

Jobs That Still Need Digi Uniform Sprites:
- Maybe Psych

Job Uniforms That Have Bad or No Implementation of Digi Sprites:
- Artist (not working)
- Mining Medic (bad sprites)

# Wiki Documentation

As far as I am aware, this shouldn't require changing anything in the wiki, but I could be wrong.

# Changelog

:cl:  
imageadd: Readded icons from ThatGuyPhil's "uniform sprites for digitigrade lizardpeople" #11017 and added/updated some
experimental: Added support for digitigrade uniforms
/:cl:

### CODE REVIEW

**Review by JamieD1 — APPROVED** [review]

**Comment by nmajask** [comment]
I think I finally figured out what the issue with the issue is with the footwraps, it appears that their icons are a bit too small and don't fully cover digi feet and, by extension, the jumpsuit.

**Comment by nmajask** [comment]
Wait, why does my almost exclusively art PR not require a review from the art director?

**Comment by nmajask** [comment]
Oh, I was still working on stuff for this and waiting for Grimm's feedback

**Pull request merged**
